bigtable-dataflow-parent/bigtable-dataflow-import/src/main/java/com/google/cloud/bigtable/dataflowimport/HBaseExportJob.java

```diff
@@ -172,14 +182,15 @@ public void processElement(ProcessContext c) throws Exception {
       List<? extends BoundedSource<T>> splitSources = source.splitIntoBundles(
           DESIRED_SHARD_SIZE, c.getPipelineOptions()
       );
+      totalShardCount.addValue((long) splitSources.size());
       LOG.info("Split into {} shards", splitSources.size());
 
       // Assign the filenames now so that the output is stable
       final List<ExportShard<T>> shards = new ArrayList<>(splitSources.size());
       int i = 0;
       for (BoundedSource<T> splitSource : splitSources) {
         final String filename = IOChannelUtils.constructName(
-            "part", ShardNameTemplate.INDEX_OF_MAX, "", i++, splitSources.size()
+            "part", ShardNameTemplate.INDEX_OF_MAX, "", ++i, splitSources.size()
```

Comment:
I thought the zero based indexing was weird too. But its how FileBasedIO does it & I wanted to stay consistent